src/Models/AZ3166Device.ts

```diff
@@ -314,6 +321,61 @@ export class AZ3166Device extends ArduinoDeviceBase {
             'Configure Device connection string completely.');
         return true;
       }
+    } else if (configSelection.detail === 'Config DPS') {
+      const option: vscode.InputBoxOptions = {
+        value:
+            'DPSEndpoint=global.azure-devices-provisioning.net;IdScope=<Id Scope>;DeviceId=<Device Id>;SymmetricKey=<Symmetric Key>',
+        prompt: `Please input DPS credentials here.`,
+        ignoreFocusOut: true,
+        validateInput: (deviceConnectionString: string) => {
+          if (!deviceConnectionString) {
+            return 'Please provide a valid DPS credentials.';
+          }
+
+          if ((deviceConnectionString.indexOf('DPSEndpoint') === -1) ||
+              (deviceConnectionString.indexOf('IdScope') === -1) ||
+              (deviceConnectionString.indexOf('DeviceId') === -1) ||
+              (deviceConnectionString.indexOf('SymmetricKey') === -1)) {
+            return 'The format of the DPS credentials is invalid.';
+          }
+          return;
+        }
+      };
+
+      const deviceConnectionString = await vscode.window.showInputBox(option);
+
+      if (!deviceConnectionString) {
+        return false;
+      }
+
+      console.log(deviceConnectionString);

```

Comment:
I think we don't need to print the connection string here for security consideration.